alacritty_terminal/src/grid/mod.rs

```diff
@@ -543,6 +600,9 @@ impl<T> Grid<T> {
     }
 
     pub fn clear_history(&mut self) {
+        // Explicitly purge all lines from history
+        let shrinkage = self.raw.len() - self.lines.0;
+        self.raw.shrink_lines(shrinkage);
```

Comment:
This fixes a separate bug, right? Need to remember that when writing a change log entry.

Reply:
@chrisduerr I don't think I was trying to fix a bug with this so much as grouping it so I could reuse it more easily.